Nos/Service/FeatureFlags.swift

```diff
@@ -22,13 +30,19 @@ class DefaultFeatureFlags: FeatureFlags, DependencyKey {
 
     private init() {}
 
-    private(set) var newMediaDisplayEnabled = false
-}
+    /// A dictionary to store the values of feature flags.
```

Comment:
Suggestion: You don't need to mention the type in the doc comment.
```suggestion
    /// Feature flags and their values.
```